abi3audit/_audit.py

```diff
@@ -21,13 +23,13 @@ class AuditError(Exception):
     pass
 
 
-@dataclass(frozen=True, eq=True, slots=True)
+@dataclass(frozen=True, eq=True)
 class AuditResult:
     so: SharedObject
     baseline: PyVersion
     computed: PyVersion
     non_abi3_symbols: set[Symbol]
-    future_abi3_objects: set[Function | Data]
+    future_abi3_objects: set[Union[Function, Data]]
```

Comment:
FWIW the `|` syntax for `Union` should also work in deferred annotations with `__future__` (it just doesn't work work in type aliases because those aren't deferred).

Reply:
I see